sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -552,18 +553,26 @@ public static Read read() {
         .setBigQueryServices(new BigQueryServicesImpl())
         .setParseFn(parseFn)
         .setMethod(Method.DEFAULT)
+        .setFormat(DataFormat.AVRO)
         .build();
   }
 
   @VisibleForTesting
-  static class TableRowParser implements SerializableFunction<SchemaAndRecord, TableRow> {
+  public static class TableRowParser implements SerializableFunction<SchemaAndRecord, TableRow> {
```

Comment:
I'm not clear on why this needs to be `public` now. It looks like its still only used in tests that are in the same package. Can't it stay package-private?

Reply:
I agree, looks like I have changed it during the prototyping stage and forgot to revert it back.